### DISCUSSION TITLE
Phoss SMP server upgrade

### DISCUSSION THREAD

**Discussion by skizub21** [phax/phoss-smp#247] [General]
Hi,

We have SMP server 5.1.2 in production planning to upgrade it to latest version available - 7.0.0

Are there any known issues related to such migration?
Is there any step by step upgrade documentation available?

**Comment by phax** [comment]
Please see the https://github.com/phax/phoss-smp/wiki/Migrations page in the Wiki and follow the steps from 5.1.x to the latest version.
Depending on the backend you used (SQL, XML, MongoDB), the changes required are different....

I recommend you follow the described steps closely :)

**Reply by skizub21** [reply]
Thank you!
